indico/modules/events/persons/controllers.py

```diff
@@ -257,47 +257,72 @@ def _process(self):
                                                has_predefined_affiliations=Affiliation.query.has_rows())
 
 
-class RHEmailEventPersons(RHManageEventBase):
+class RHEmailEventPersonsPreview(RHManageEventBase):
+    """Preview an email with EventPersons associated placeholders."""
+
+    @use_kwargs({
+        'body': fields.String(load_default=''),
+        'subject': fields.String(load_default=''),
```

Comment:
```suggestion
        'body': fields.String(required=True),
        'subject': fields.String(required=True),
```

Unless I'm mistaken, you always pass these arguments, so there is no need to allow calling this endpoint with them not being sent.